library/std/src/panic.rs

```diff
@@ -308,8 +308,7 @@ pub fn get_backtrace_style() -> Option<BacktraceStyle> {
                 BacktraceStyle::Short
             }
         })
-        .unwrap_or(if cfg!(target_os = "fuchsia") {
-            // Fuchsia components default to full backtrace.
+        .unwrap_or(if crate::sys::BACKTRACE_DEFAULT {
```

Comment:
If we want this to be a bool instead of an enum, here, we should actually give it an appropriate name.

```suggestion
        .unwrap_or(if crate::sys::FULL_BACKTRACE_DEFAULT {
```